netbox/virtualization/views.py

```diff
@@ -172,7 +173,17 @@ class ClusterView(generic.ObjectView):
     queryset = Cluster.objects.all()
 
     def get_extra_context(self, request, instance):
-        return instance.virtual_machines.aggregate(vcpus_sum=Sum('vcpus'), memory_sum=Sum('memory'), disk_sum=Sum('disk'))
+        vm_memory = [convert_byte_size(item.memory, 'mega') for item in instance.virtual_machines.all() if item.memory]
```

Comment:
It's sufficient to capture and then round the raw aggregate value. Also, we generally want to handle this sort of massaging of numbers in the template (via a template tag) rather than in the view to afford maximum flexibility.